concent_api/core/tests/test_integration_subtask_results_verify.py

```diff
@@ -50,6 +50,8 @@ def setUp(self):
             task_id=self.task_id,
         )
         self.report_computed_task = self._create_report_computed_task()
+        self.scene_file_without_golem_path = self.report_computed_task.task_to_compute.compute_task_def['extra_data'][
```

Comment:
Why is this replaced here ? Shouldn't it be replaced in `send_blender_verification_request` ?

Reply:
I mean that it is already being done there now https://github.com/golemfactory/concent/pull/734/files#diff-6f9c043a7cc2560c4e15c0127a57dc25R29. Why do you have to replace it before sending request ?

Reply:
After discussion, please extract code for replacing beginning of the path to separate function and reuse in both places.